libs/sdk-ui-charts/src/highcharts/adapter/plugins/renderBubbles.ts

```diff
@@ -65,111 +60,5 @@ export function renderBubbles(HighchartsInstance: any): void {
                 return radius;
             },
         );
-
-        // #SD-479 fix bubbles is not rendered with min/max config
-        wrap(HighchartsInstance.Axis.prototype, "beforePadding", function (_proceed: any) {
```

Comment:
This method beforePadding does not exist in current version. 
Tested with min max config bubbles rendered well. 
So delete this unused code.